src/HotTable.vue

```diff
@@ -32,7 +30,11 @@
   const HotTable: HotTableComponent<Vue, HotTableData, HotTableMethods, {}, HotTableProps> = {
     name: 'HotTable',
     props: propFactory('HotTable'),
-    watch: propWatchFactory(updateHotSettings),
+    watch: {
+      mergedHotSettings: function() {
+        this.hotInstance.updateSettings(this.mergedHotSettings);
```

Comment:
I would use `value` from function arguments instead of reading `this.mergedHotSettings`. Value is already read and cached. But I don't know if it makes any differences.

Reply:
Good idea, much cleaner 👍